ui-v2/app/models/service.js

```diff
@@ -16,7 +16,6 @@ export default Model.extend({
   InstanceCount: attr('number'),
   ConnectedWithGateway: attr(),
   ConnectedWithProxy: attr(),
-  Proxy: attr(),
```

Comment:
We still use this further down in the model:

https://github.com/hashicorp/consul/blob/708957a982429df1727f27a7f9dc307ac1f3d3b2/ui-v2/app/models/service.js#L61-L67

and we use the computedProperties there for sorting.

Its added during normalisation here

https://github.com/hashicorp/consul/blob/708957a982429df1727f27a7f9dc307ac1f3d3b2/ui-v2/app/serializers/service.js#L33

Reply:
oh, sorry. I removed it while rebasing this old branch.